content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -464,6 +464,68 @@ go run ./write.go
 
 <!------------------------------- END GO CONTENT ------------------------------>
 {{% /tab-content %}}
+{{% tab-content %}}
+<!------------------------------- BEGIN TELEGRAF CONTENT ------------------------------>
+Use [Telegraf](/{{< latest "telegraf" >}}/) to consume line protocol or CSV,
+and then write line protocol to {{< cloud-name >}}.
+
+1. If you haven't already, follow the instructions to [download and install Telegraf](/{{< latest "telegraf" >}}/install/).
+
+2. Copy and save the [home sensor data sample](#home-sensor-data-line-protocol) to a file on your local system--for example, `home.lp`.
+
+3. Run the following command to generate a Telegraf configuration file (`./telegraf.conf`) that enables the `inputs.file` and `outputs.influxdb_v2` plugins:
+
+    ```sh
+    telegraf --sample-config \
+      --input-filter file \
+      --output-filter influxdb_v2 \
+      > telegraf.conf
+    ```
+
+4. In your editor, open `./telegraf.conf` and do the following:
+
+  - In the `[[inputs.file]].files` list, replace `"/tmp/metrics.out"` with your sample data filename. If Telegraf can't find a file when started, it stops processing and exits.
+
+    ```toml
+    [[inputs.file]]
+      ## Files to parse each interval.  Accept standard unix glob matching rules,
+      ## as well as ** to match recursive files and directories.
+      files = ["home.lp"]
+    ```
+  - In the `[[outputs.influxdb_v2]]` section, replace the default values with the following configuration for your InfluxDB Cloud Dedicated cluster:
+
+    ```toml
+    [[outputs.influxdb_v2]]
+
+      # InfluxDB Cloud Dedicated cluster URL
+      urls = ["https://cluster-id.influxdb.io"]
+
+      # INFLUX_TOKEN is an environment variable you assigned to your database token
+      token = "${INFLUX_TOKEN}"
+
+      # Leave organization empty for InfluxDB Cloud Dedicated
+      organization = ""
+
+      # Replace DATABASE_NAME with your InfluxDB Cloud Dedicated database name
+      bucket = "DATABASE_NAME"
+    ```
+
+    Replace the following:
+
+    - **`DATABASE_NAME`**: your InfluxDB Cloud Dedicated [database](/influxdb/cloud-dedicated/admin/databases/)
+
+5. To write the data, execute the `telegraf` daemon with the following options:
+
+    - `--config`: Specifies the filepath of the configuration file.
+    - `--once`: Runs a single Telegraf collection cycle for the configured inputs and outputs, and then exits.
+
+    Enter the following command in your terminal:
+
+    ```sh
+    telegraf --once --config ./telegraf.conf
+    ```
+<!------------------------------- END TELEGRAF CONTENT ------------------------------>
```

Comment:
Updated Telegraf get started content looks great! Just need to port it to the Get started docs in serververless.